### DISCUSSION TITLE
Announcements - March 2021

### DISCUSSION THREAD

**Discussion by ghobona** [opengeospatial/developer-track#6] [Announcements]
Announcements will be made in this channel.

**Comment by ghobona** [comment]
All, 

Anyone planning to do a demo should register their demo by 17:30 EDT (21:30 GMT) at https://github.com/opengeospatial/developer-track/blob/main/demonstrations.adoc

Simply create a Pull Request and we will merge it into the document.

**Reply by ghobona** [reply]
10 minute extension for registration of demos. 

Registration will close at 17:40 EDT.

**Comment by ghobona** [comment]
Until 17:15 EDT on March 25th 2021, you can vote for your favorite demo at https://twitter.com/opengeospatial

**Reply by tomkralidis** [reply]
Hi all: I had to shift to another meeting (OGC MetOceanDWG) during our demo/discussion and realize I may not have muted myself -- apologies for any inconvenience that this may have caused!
